packages/apps/ability-aave/src/lib/validateSimulation.ts

```diff
@@ -0,0 +1,66 @@
+import type { ValidateSimulationParams } from '@lit-protocol/vincent-ability-sdk/gatedSigner';
+
+import { getAddress, zeroAddress } from 'viem';
+
+import { getAaveAddresses, getATokens } from './helpers/aave';
+
+export const validateSimulation = (params: ValidateSimulationParams) => {
+  const { chainId, sender: _sender, simulation } = params;
```

Comment:
it's really nice that we do the simulation for them and just provide it.  love this.